vgrep.go

```diff
@@ -41,10 +44,12 @@ var (
 	Matches [][]string
 	Log     = logrus.New()
 	version string // set in the Makefile
+	lock    lockfile.Lockfile
```

Comment:
Can we rename it to `Lock`? It has zero impact but the other global variables are exported as well to make it more obvious that they are package global.